core/src/main/java/org/jdbi/v3/core/mapper/MapMapper.java

```diff
@@ -28,21 +32,26 @@
  * default.
  */
 public class MapMapper implements RowMapper<Map<String, Object>> {
-    private final boolean foldCase;
+    /**
+     * @deprecated TODO remove in jdbi4
+     */
+    @Deprecated
+    private final Boolean toLowerCase;
 
     /**
      * Constructs a new MapMapper, with map keys converted to lowercase.
      */
     public MapMapper() {
-        this(true);
+        this(null);
     }
 
     /**
      * Constructs a new MapMapper
-     * @param foldCase if true, column names are converted to lowercase in the mapped {@link Map}.
+     * @param toLowerCase if true, column names are converted to lowercase in the mapped {@link Map}. If null, the new MapNappers config will be used.
      */
-    public MapMapper(boolean foldCase) {
-        this.foldCase = foldCase;
+    // TODO deprecate when MapMappers.caseChange is out of beta
+    public MapMapper(@Nullable Boolean toLowerCase) {
```

Comment:
Changing this from `boolean` to `Boolean` is a binary breaking change.

Let's change this back to `boolean`, and let the other constructor explicitly assign null rather than delegating to this constructor.